tests/fixtures/sample-with-diagram.adoc

```diff
@@ -14,3 +14,119 @@ Thanks to link:https://github.com/ryosms[]!
 class Sample {
 }
 ----
+
+
+= Additional Diagrams
```

Comment:
Can you remove this line ? It is giving this error when running manually
```
bash-4.3# asciidoctor -r asciidoctor-diagram sample-with-diagram.adoc 
asciidoctor: ERROR: sample-with-diagram.adoc: line 19: only book doctypes can contain level 0 sections
```

Reply:
I could make it a == instead of =. 
That way the doctype is satisfied.

Reply:
@fnchooft it would be great. Thank you!